### PR TITLE
oatpp-postgresql: add version 1.3.0

### PR DESCRIPTION
**Pull request by toge**
Specify library name and version:  **oatpp-postgresql/1.3.0**

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by ericLemanissier — DISMISSED** [review]

**Review by madebr — APPROVED** [review]

File: recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index b538d15..7c811a8 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -47,6 +47,10 @@ target_link_libraries(${OATPP_THIS_MODULE_NAME}
+         PUBLIC ${PostgreSQL_LIBRARIES}
+ )
+ 
++if(MSVC)
```
Comment (madebr) [inline recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch:9]: Can this patch be upstreamed?

**Review by madebr — APPROVED** [review]

File: recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch (hunk 1 above)
Comment (toge) [reply recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch:9]: I will create PR for this.

**Review by ericLemanissier — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch (hunk 1 above)
Comment (toge) [reply recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch:9]: merged.
https://github.com/oatpp/oatpp-postgresql/pull/15

Comment (madebr) [reply recipes/oatpp-postgresql/all/patches/1.3.0/01-add-bigobj.patch:9]: If it's good enough for upstream, it's more then okay for us 🤣

**Pull request merged**